src/main/java/org/prebid/server/proto/openrtb/ext/request/amx/ExtImpAmx.java

```diff
@@ -0,0 +1,19 @@
+package org.prebid.server.proto.openrtb.ext.request.amx;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+@AllArgsConstructor(staticName = "of")
+@Value
+public class ExtImpAmx {
+
+    @JsonProperty("tagId")
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
```

Comment:
Don't think we really want to add additional logic for field representing in POJO models.
Let's keep it as simple as possible.
In any case bidder's implementation should care.